lib/db/postgres/native/connection.rb

```diff
@@ -186,6 +186,29 @@ def send_query(statement)
 					flush
 				end
 				
+				def send_query_params(statement, *params)
+					params = params.map(&:to_s)
+					size = params.size
+					is_binary = params.map { |param| param.include?("\x00".b) }
+					
+					# type 17 => bytea (https://github.com/postgres/postgres/blob/master/src/include/catalog/pg_type.dat)
+					paramTypes = FFI::MemoryPointer.new(:int, params.size)
+					paramTypes.write_array_of_type(FFI::Type::INT, :put_int, is_binary.map { |binary| binary ? 17 : 0 })
+					
+					paramValues = FFI::MemoryPointer.new(:pointer, params.size)
+					paramValues.write_array_of_pointer(params.map { |param| FFI::MemoryPointer.from_string(param) })
```

Comment:
I'm not sure if this is GC safe, or if we need an additional variable to save the the result of the `params.map` operation.